packages/astro/package.json

```diff
@@ -108,6 +108,7 @@
     "dev": "astro-scripts dev --copy-wasm --prebuild \"src/runtime/server/astro-island.ts\" --prebuild \"src/runtime/client/{idle,load,media,only,visible}.ts\" \"src/**/*.{ts,js}\"",
     "postbuild": "astro-scripts copy \"src/**/*.astro\" && astro-scripts copy \"src/**/*.wasm\"",
     "test": "pnpm run test:node",
+    "test:match": "pnpm run test:node --match",
```

Comment:
`node:test`'s `match` is unfortunately terrible, it takes multiple minutes on my computer since it runs through every file still. However, the command in the root repo was broken, so this fixes that.